.github/workflows/python-integration.yml

```diff
@@ -29,10 +25,10 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.8", "3.9", "3.10", "3.11"]
+        test-run: [ "S3", "S3 Express"]
```

Comment:
Could we use paired values like described in this StackOverflow answer? https://stackoverflow.com/a/68940067

Reply:
Thanks a lot! That's awesome, I looked for something similar, but didn't manage to find it.